.env.production

```
@@ -0,0 +1,14 @@
+NODE_ENV=production
+DATABASE_URL=postgresql://neondb_owner:npg_AGR4shun0lLM@ep-icy-bird-a9lng7lk-pooler.gwc.azure.neon.tech/neondb?sslmode=require&channel_binding=require
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DATABASE_URL key should go before the NODE_ENV key

(UnorderedKey)